code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -142,74 +149,79 @@ Credit where due:
+		return
+	L.set_species(/datum/species/human)
+	L.equipOutfit(/datum/outfit/servant_of_ratvar)
+	return TRUE
+
+/datum/game_mode/proc/open_portals_to_reebe()
+	priority_announce("Massive spacetime anomalies appearing across the station. All personnel are hereby directed to enter the anomalies and neutralize the source. This is not a drill.", "Spacetime Anomaly Alert", 'sound/magic/clockwork/gateways_open.ogg')
+	if(get_security_level() != "delta")
+		set_security_level("red")
+	var/obj/effect/landmark/L
+	var/turf/T
+	for(var/V in GLOB.generic_event_spawns)
+		L = V
+		T = get_turf(L)
+		addtimer(CALLBACK(src, .proc/open_portal, T), rand(50, 1200))
+	var/obj/structure/destructible/clockwork/massive/celestial_gateway/C = GLOB.ark_of_the_clockwork_justiciar
+	C.spawn_animation() //get this party started!
+
+/datum/game_mode/proc/open_portal(turf/portal_loc)
+	var/obj/effect/clockwork/reebe_rift/R = new(portal_loc)
+	R.visible_message("<span class='warning'>The air above [portal_loc] screeches and shimmers as a portal appears!</span>")
+	playsound(portal_loc, 'sound/effects/supermatter.ogg', 50, 0)
+
+/datum/game_mode/clockwork_cult/check_finished()
+	if((SSshuttle.emergency.mode == SHUTTLE_ENDGAME))
 		return TRUE
-	return FALSE
+	if(GLOB.clockwork_gateway_activated || !GLOB.ark_of_the_clockwork_justiciar)
+		return TRUE
+	..()
 
 /datum/game_mode/clockwork_cult/proc/check_clockwork_victory()
-	if(GLOB.clockwork_gateway_activated)
-		SSticker.news_report = CLOCK_PROSELYTIZATION //failure, technically, but we have the station
-		if(GLOB.ratvar_awakens)
-			SSticker.news_report = CLOCK_SUMMON
-			return TRUE
-	else
-		SSticker.news_report = CULT_FAILURE
-	return FALSE
+	if(GLOB.clockwork_gateway_activated || GLOB.ark_of_the_clockwork_justiciar)
+		SSticker.news_report = CLOCK_SUMMON
+		return TRUE
+	SSticker.news_report = CLOCK_FAILURE
+	return
 
 /datum/game_mode/clockwork_cult/declare_completion()
-	..()
-	return 0 //Doesn't end until the round does
+	var/text = ""
+	var/snd
+	if(check_clockwork_victory())
+		if(SSshuttle.emergency.mode != SHUTTLE_ENDGAME)
+			text += "<span class='large_brass'><b>Ratvar's servants defended the Ark and summoned Ratvar!</b></span>"
+			snd = sound('sound/ambience/antag/ClockCultAlr.ogg', volume = 50)
+			SSticker.mode_result = "win - servants defended the ark"
+		else
+			text += "<span class='large_brass'><b>The crew ran away like wimps, allowing Ratvar to rise unopposed!</b></span>"
+			snd = sound('sound/ambience/antag/ClockCultAlr.ogg', volume = 50)
+			SSticker.mode_result = "win - crew fled"
+	else
+		text += "<span class='userdanger'>The Ark has been destroyed! Ratvar will rust away in Reebe for all eternity!</span>"
+		snd = sound('sound/magic/clockwork/ratvar_attack.ogg', volume = 50)
+		SSticker.mode_result = "loss - ark destroyed"
+	to_chat(world, text)
+	to_chat(world, snd) //Yes, this actually works
```

Comment:
Don't use `to_chat` to send sounds. Use [`playsound_global`](https://github.com/tgstation/tgstation/blob/master/code/game/sound.dm#L134)